Add quantized ops to pybindings

lucylq opened the pull request.

larryliu0820 commented on examples/models/llama2/runner/generation.py:
```diff
@@ -19,9 +19,20 @@
 from executorch.extension.pybindings.portable_lib import _load_for_executorch
 
 
+from executorch.kernels import quantized
```
import executorch.kernels.quantized  # noqa[F401] 'executorch.kernels.quantized' imported but unused

larryliu0820 commented on .ci/scripts/test_pybind.py:
```diff
@@ -0,0 +1,52 @@
+import torch
```
For this test, can we do the following things:

* Add it into the other test modules https://github.com/pytorch/executorch/tree/main/extension/pybindings/test
* Use `unittest` framework
* Add a test for exporting an nn.Module with quantized ops to executorch, read it back using pybind and run it.

lucylq replied:
Ah, thanks

lucylq replied:
Referred to https://github.com/pytorch/executorch/blob/main/exir/tests/test_quant_fusion_pass.py#L40 for quant test.

larryliu0820 commented on extension/pybindings/test/test_pybindings.py:
```diff
@@ -13,6 +13,15 @@
     )
 
     kernel_mode = "portable"
+    # Load portable lib.
```
Would be good to mention that we have to load portable_lib first

lucylq committed:
Add quantized ops to pybindings (#4696)

Summary: Pull Request resolved: https://github.com/pytorch/executorch/pull/4696

Reviewed By: larryliu0820

Differential Revision: D61308890

Pulled By: lucylq
